packages/trader/src/Modules/Trading/Containers/trade-params.tsx

```diff
@@ -37,7 +40,7 @@ const TradeParams = observer(({ is_minimized }) => {
                     {isVisible('barrier_selector') && <BarrierSelector key={'barrier_selector'} />}
                 </Fieldset>
             )}
-            {isVisible('amount') && <Amount key={'amount'} is_minimized={is_minimized} />}
+            {isVisible('amount') && <Amount key={'amount'} is_minimized={is_minimized as boolean} />}
```

Comment:
@henry-deriv
```suggestion
            {isVisible('amount') && <Amount key={'amount'} is_minimized={is_minimized} />}
```
it's better to make this prop optional in packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.tsx :)

Reply:
In 4th package `is_minimized` has default value (`false`), it can also fix this issue